### DISCUSSION TITLE
Native Emoji picker for editor

### DISCUSSION THREAD

**Discussion by OCram85** [outline/outline#4239] [Ideas]
It would be awesome if the editor could suggest Emojies while typing `:` like github or gitea...

![image](https://user-images.githubusercontent.com/13993700/194304662-c49a3fea-75c9-4ef7-9e73-cbd3b1fb951f.png)

![image](https://user-images.githubusercontent.com/13993700/194304710-09d71d72-e968-4ce9-b1d3-7206b1179378.png)

> 💡 Note: This would help especially windows users, because the build-in Emoji selection triggered by `<ctrl> + .` is awefull. You have to guess the emoji name translation for the current os language.

**Comment by tommoor** [comment]
We already do this?

<img width="351" alt="image" src="https://user-images.githubusercontent.com/380914/194305594-5b2a7caf-7c6b-4da1-a80e-bbf9ec9fb5fd.png">

**Reply by OCram85** [reply]
Sorry my mistake... Tested it with multiple browser and forgot disabling the JS blockers ^^

**Reply by OCram85** [reply]
@tommoor: when is the popup supposed to appear?
I noticed only shows  up after typing for example `:heart` and then removing chars with <kbd>backspace</kbd>, if you're using it in a heading line

**Reply by tommoor** [reply]
It’s active in all circumstances that you can type text, above screenshot in a paragraph.
